force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -300,15 +355,20 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static Map<String, Integer> addItemsToMap() {
         // Initialize an empty Map
-
+        Map<String, Integer> fruits =  new Map<String, Integer> ();
         // Add key-value pairs to the Map
         // add Apples => 5
+        fruits.put('Apples', 5);
         // add Oranges => 10
+        fruits.put('Oranges', 10);
         // add Bananas => 15
+        fruits.put('Bananas', 15);
         // add Pears => 20
+        fruits.put('Pears', 20);
         // add Grapes => 25
+        fruits.put('Grapes', 25);
 
-        return null; // Replace null with the variable you used to store the result
+        return fruits; // Replace null with the variable you used to store the result
```

Comment:
Good job. Another way to populate a map when you know all the values is to use the Map constructor. Its syntax is a bit funny compared to the rest of Apex. 

```
 public static Map<String, Integer> addItemsToMap() {

        return new Map<String, Integer>{
                'Apples' => 5,
                'Oranges' => 10,
                'Bananas' => 15,
                'Pears' => 20,
                'Grapes' => 25
        };
    }
 ```